types/redux-state-sync/redux-state-sync-tests.ts

```diff
@@ -18,10 +23,14 @@ const middleware = createStateSyncMiddleware({
 // $ExpectError
 const middlewareError = createStateSyncMiddleware({ broadcastChannelOption: null });
 
-function rootReducer(state: TestState, action: Action): TestState {
+function rootReducer(state: TestState = initialState, action: Action): TestState {
     return state;
 }
 
-const store = createStore(withReduxStateSync(rootReducer, (state) => state), ['test'], applyMiddleware(middleware));
+const store = createStore(withReduxStateSync(rootReducer, (state) => state), initialState, applyMiddleware(middleware));
 initStateWithPrevTab(store);
 initMessageListener(store);
+store.getState().a;
+store.getState().b;
+// $ExpectError
+store.getState().missingProperty;
```

Comment:
Before these tests, nothing detected that `getState()` return type was a reducer function, instead of TestState